custom_components/tuya_local/devices/dry_pury_evo_dehumidifier.yaml

```diff
@@ -0,0 +1,160 @@
+        - dps_val: sleep
+          value: sleep
+          icon: "mdi:power-sleep"
+          icon_priority: 3
+    - id: 6
+      name: current_humidity
+      type: integer
+secondary_entities:
+  - entity: fan
+    dps:
+      - id: 1
+        name: switch
+        type: boolean
+      - id: 5
+        name: dehumidifier_mode
+        type: string
+        hidden: true
+      - id: 4
+        name: speed
+        type: string
+        mapping:
+          - dps_val: low
+            value: 50
+            constraint: dehumidifier_mode
+            conditions:
+              - dps_val: laundry
+                invalid: true
+          - dps_val: high
+            value: 100
+            constraint: dehumidifier_mode
+            conditions:
+              - dps_val: sleep
+                invalid: true
+  - entity: sensor
+    name: Current humidity
+    # deprecated: humidifier      # without that I see humidity values and it's history
+    # category: diagnostic        # without that I see humidity values and it's history
+    class: humidity
+    dps:
+      - id: 6
+        name: sensor
+        type: integer
+        precision: 0
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    class: temperature
+    dps:
+      - id: 7
+        name: sensor
+        type: integer
+        unit: C
+        class: measurement
+  - entity: switch
+    name: Ionizer
+    icon: "mdi:creation"
+    optional: true
+    dps:
+      - id: 10
+        type: boolean
+        name: switch
+  - entity: lock
+    name: Child lock
+    category: config
+    dps:
+      - id: 16
+        type: boolean
+        name: lock
+        mapping:
+          - dps_val: false
+            icon: "mdi:hand-back-right"
+          - dps_val: true
+            icon: "mdi:hand-back-right-off"
+  - entity: select
+    name: Timer
+    category: config
+    dps:
+      - id: 17
+        type: string
+        name: option
+        mapping:
+          - dps_val: "cancel"
+            value: Continuous
+          - dps_val: 1h
+            value: 1 hour
+          - dps_val: 2h
+            value: 2 hours
+          - dps_val: 3h
+            value: 3 hours
+  - entity: binary_sensor # based on API doc. I don't know how to test
+    name: Fault
+    class: problem
+    category: diagnostic
+    dps:
+      - id: 19
+        name: sensor
+        type: bitfield
+        mapping:
+          - dps_val: 0
+            value: E1
+          - dps_val: 1
+            value: CL
+          - dps_val: 2
+            value: CH
+          - dps_val: 3
+            value: LO
+          - dps_val: 4
+            value: FL
+          - dps_val: 5
+            value: Cleaning
+  - entity: button # based on API doc. Nothing happened when I run on my device
+    name: Filter reset
+    category: config
+    optional: true
+    dps:
+      - id: 20
+        type: boolean
+        name: button
+  - entity: select # based on API doc. Nothing happened when I run on my device
```

Comment:
If it does not do anything, and the device is not reporting it, then just leave it out of the config.